README.md

```diff
@@ -101,6 +101,14 @@ set :passenger_environment_variables, { rbenv_version: '3.0.1' }
 
 If you are running passenger in standalone mode, it is possible for you to put passenger in your Gemfile and rely on capistrano-bundler to install it with the rest of your bundle.  If you are installing passenger during your deployment **AND** you are using the new restart method (see below), you need to set `:passenger_in_gemfile` to `true` in your `config/deploy.rb`.
 
+### Note for asdf users
+
+If you are using standalone passenger, and asdf through the [capistrano-asdf gem](https://github.com/cabesa-collective/capistrano-asdf), it should be enough to add the passenger binaries to the asdf variable.
+
+``` ruby
+set :asdf_map_ruby_bins, %w[rake bundle gem ruby passenger passenger-config]
```

Comment:
This can be automated.  Notice how we're detecting what other passenger plugins you are using and adding `passenger` to the lists of bin appropriately:

https://github.com/capistrano/passenger/blob/master/lib/capistrano/tasks/passenger.cap#L98-L107